com.gitlab.davem.ClamTk.yaml

```diff
@@ -92,25 +89,26 @@ modules:
   - name: clamtk
     buildsystem: simple
     build-commands:
-      - 'install -Dm 755 clamtk /app/bin/clamtk'
+      - 'install -Dm 755 clamtk ${FLATPAK_DEST}/bin/clamtk'
       - |
         sitelib=$(perl -e 'use Config; print $Config{installsitelib};')
         install -d $sitelib
         cp -r lib $sitelib/ClamTk
-      - 'install -Dm 644 clamtk.desktop /app/share/applications/com.gitlab.davem.ClamTk.desktop'
-      - 'install -Dm 644 images/clamtk.png /app/share/icons/hicolor/128x128/apps/clamtk.png'
-      - 'install -Dm 644 com.gitlab.davem.ClamTk.metainfo.xml -t /app/share/metainfo'
+      - 'sed -i "s/Icon=clamtk/Icon=com.gitlab.davem.ClamTk/g" clamtk.desktop'
+      - 'install -Dm 644 clamtk.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
+      # Preferences depend on the file `clamtk.png`
+      - 'install -Dm 644 images/clamtk.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/clamtk.png'
+      - 'install -Dm 644 images/clamtk.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png'
+      - 'install -Dm 644 com.github.davetheunsub.clamtk.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml'
```

Comment:
Should leave the name as `metainfo.xml`, Flatpak will create an `appdata.xml` alias itself.

Reply:
Where? `com.github.davetheunsub.clamtk.metainfo.xml` doesn't exist, the upstream name is appdata.xml https://github.com/dave-theunsub/clamtk/blob/master/com.github.davetheunsub.clamtk.appdata.xml

Reply:
Hm, now it fails with appdata.xml https://buildbot.flathub.org/#/builders/6/builds/3964, the previous name was correct. Not sure what happened there.